StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1007UnitTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace StyleCop.Analyzers.Test.SpacingRules
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using StyleCop.Analyzers.SpacingRules;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// This class contains unit tests for <see cref="SA1007OperatorKeywordMustBeFollowedBySpace"/> and
+    /// <see cref="SA1007CodeFixProvider"/>.
+    /// </summary>
+    public class SA1007UnitTests : CodeFixVerifier
+    {
+        [Fact]
+        public async Task TestOperatorKeywordCases()
```

Comment:
Its not a big deal. Your test pretty covers everything important. I think this can be merged in.